common/cpp/tcf_error.h

```diff
@@ -45,6 +45,16 @@ typedef enum {
     TCF_ERR_INVALID_WORKLOAD = -12
 } tcf_err_t;
 
+/** Avalon worker enclave Type (singleton, KME, WPE) */
+enum EnclaveType {
+    // SINGLETON_ENCLAVE does both Key management and workload processing
```

Comment:
Doxygen requires one of these formats so it will know to use it in documentation:
`/// SINGLETON_ENCLAVE does both Key management and workload processing`
`//! SINGLETON_ENCLAVE does both Key management and workload processing`
`/** SINGLETON_ENCLAVE does both Key management and workload processing */`

Use one of the above here and on lines 52 and 54.

See 
http://www.doxygen.nl/manual/docblocks.html#cppblock